shared/elements/gaia_grid/js/grid_dragdrop.js

```diff
@@ -666,6 +672,12 @@
               return;
             }
 
+            if (this.gridView._launchingApp) {
```

Comment:
It should be reliable, but if it's easier to leave as-is, I'm fine with that. Thanks for looking.